comparators/modified_survey_point.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = modifiedSurveyPoint;
+
+function modifiedSurveyPoint(newVersion, oldVersion) {
+  if (
+    !newVersion.deleted &&
+    newVersion.properties['osm:version'] !== 1 &&
```

Comment:
Hi @willemarcel , we have a basic fixture https://github.com/mapbox/osm-compare/blob/master/tests/basicFixture.json which always gets tested against all the compare functions, in that it also have fixtures where either of oldversion/newversion is null. Therefore we always make sure the following checks before using them:
- oldVersion
- newVersion
- oldVersion.properties
- newVersion.properties
- oldVersion.geometry
- newVersion.geometry